### DISCUSSION TITLE
Last update on CPU Usage

### DISCUSSION THREAD

**Discussion by ceciherr** [psm-compute/psm-compute.github.io#17] [General]
Please note that this command doesn't actually show how many CPUs are in use, but rather how many are currently being requested. It counts simulations in the 'waiting' state (W) as well as those that are actively 'running' (R).

**Comment by simongravelle** [comment]
@ceciherr I am not sure what you are referring too here ? which command ?

edit: ah yes, of course, the "Measure the CPU Usage per User" section. Yes I did that a bit too quickly

**Comment by simongravelle** [comment]
@ceciherr PR #18 should help

**Reply by ceciherr** [reply]
Indeed PR #18 fixes it, thanks for having a look
